corehq/apps/reports/standard/deployments.py

```diff
@@ -513,6 +513,15 @@ def get_chart_percent_series(self):
             def get_buckets(self):
                 return self.bucket_series.get_summary_data()
 
+            @property
+            def help(self):
+                return _(
+                    "<strong>Aggregate Percents</strong> shows the percent of users who have {0} "
+                    "<em>since</em> a certain date.<br><br>"
+                    "<strong>Daily Counts</strong> shows the count of users who <em>last {0} on</em> "
+                    "that particular day."
+                ).format(self.verb)
```

Comment:
It's nice to cut down on the almost duplicate strings, but this might not be translatable.  Depends on how cleanly "submitted/synced" can be substituted in in French or Spanish.  It'd probably be safer to just copy/paste the string.

Reply:
good point, thanks. updated.